tbx/project_styleguide/templates/patterns/styleguide/typography/typography.html

```diff
@@ -1,100 +1,106 @@
-<div class="p-4 mb-8" style="max-width: 710px">
-    {# Headings #}
-    <p class="underline mt-8">Heading 1</p>
-    {% include "patterns/atoms/motif-heading/motif-heading.html" with heading_level=1 classes="motif-heading--one" %}
+{% extends "patterns/styleguide_base.html" %}
```

Comment:
Nice idea creating this so the we can test the typography with the different themes as well.